### DISCUSSION TITLE
Help! I'm struggling to display activities associated to my custom object

### DISCUSSION THREAD

**Discussion by dg123123** [deejay-hub/timeline-lwc#199] [Q&A] [answered]
I've got a custom object called "Applications" and we're using native Activities associated to the Applications records.

I would like to display the related Activities on the timeline. I used the existing custom metadata records as a "template" to create my own entry, as shown in the attached screenshot...but it's not working.

What am I missing?

![image](https://user-images.githubusercontent.com/107086809/172509421-5a82cf43-2045-4f48-878e-a8ef5eaa047f.png)

Thanks in advance!

**Comment by deejay-hub** [comment, marked as answer]
Looks like the 'Relationship Name' field value is incorrect. Since you're using the standard Tasks you can use the standard relationship that is auto created on all custom and standard objects. In short .. you do not need the __r. So the value should be just **Tasks**.

**Reply by dg123123** [reply]
Thank you so much!!
